src/docker-utils.coffee

```diff
@@ -101,27 +101,30 @@ do ->
 		Promise.using readLockImages(), ->
 			dockerProgress.pull(image, onProgress)
 
-	supervisorTag = config.supervisorImage
```

Comment:
(The getRegistryAndName can throw, so it does make sense that it's wrapped as a Promise)